warden/integration-tests/action.yml

```diff
@@ -0,0 +1,183 @@
+name: "Integration Tests Mage-OS in Warden"
+author: "Vladyslav Podorozhnyi"
+description: "A Github Action that run Integration Tests of Mage-OS in warden."
+
+inputs:
+  search:
+    required: true
+    default: "elasticsearch:7.15.1"
```

Comment:
I would of thought elasticsearch, but I'm not overly opinionated on it.

Reply:
I'm also not overly opinionated, but since ES changed their licensing, my feeling is OS is the new default. [Commerce on Cloud](https://experienceleague.adobe.com/docs/commerce-operations/installation-guide/system-requirements.html?lang=en) only supports OS by now. Many hosters do not provide any newer ES versions any more due to the licensing change.

Reply:
Fair argument, I'll get it swapped :D